C#/MarsRoverKata/MarsRoverKata/Src/GridPosition.cs

```diff
@@ -2,15 +2,25 @@
 {
     class GridPosition
     {
-        public int X;
-        public int Y;
+        public readonly int X;
```

Comment:
Fields still not encapsulated! I thought about moving MoveNorth etc, to the GridPosition class but not sure it fits the domain - should I tell the GridPosition to move North or tell the Grid to move the GridPosition North?